src/components/screens/ScoreSetEditor.vue

```diff
@@ -278,6 +278,7 @@
               </template>
             </Card>
             <div v-if="itemStatus == 'NotLoaded' || this.item.private">
+              {{ scoreRanges }}
```

Comment:
I think you missed removing this from debugging.
```suggestion
```

Reply:
hahaha~~ Yes, you're correct!